trade-finance-manager-ui/server/controllers/deals/index.js

```diff
@@ -92,18 +79,58 @@ const queryDeals = async (req, res) => {
     });
   }
 
+  const activeSortByField = sortBy.field;
+  const activeSortByOrder = sortBy.order;
+  const sortButtonWasClicked = sortfield ? true : false;
```

Comment:
Seems this could just be `sortButtonWasClicked = sortfield` or maybe rename `sortfield` to `sortButtonWasClicked`